README.md

```diff
@@ -108,7 +97,7 @@ After=network.target
 After=msgbus.service
```

Comment:
Do we still need the msgbusd service? its purpose was proxying messages to zos nodes, which isn't the case anymore, all of our clients can communicate with nodes through relays